ascmonitor/mendeleur.py

```diff
@@ -48,7 +51,10 @@ def all_documents(self):
         Fetch the current library from mendeley.
         :returns: List of dicts with document bibliography
         """
-        library = self.group.documents.iter(sort="created", order="desc", view="all")
+        library = self.group.documents.iter(page_size=500, sort="created", order="desc", view="all")
```

Comment:
I sure hope the mendeley responses are deterministic? Otherwise this sounds like a lot of data..

Reply:
It is a lot of data ;-) But I guess until we reach 100k entries this wont be a problem